example/Makefile

```diff
@@ -103,4 +126,93 @@ stop-lvmd:
+		topolvm-external-provisioner-runner \
+		topolvm-external-resizer-runner \
+		topolvm-external-snapshotter-runner \
+		topolvm-system:controller \
+		topolvm-system:node
+	$(KUBECTL) delete clusterrolebinding \
+		topolvm-cert-manager-cainjector \
+		topolvm-cert-manager-controller-approve:cert-manager-io \
+		topolvm-cert-manager-controller-certificates \
+		topolvm-cert-manager-controller-certificatesigningrequests \
+		topolvm-cert-manager-controller-challenges \
+		topolvm-cert-manager-controller-clusterissuers \
+		topolvm-cert-manager-controller-ingress-shim \
+		topolvm-cert-manager-controller-issuers \
+		topolvm-cert-manager-controller-orders \
+		topolvm-cert-manager-webhook:subjectaccessreviews \
+		topolvm-csi-provisioner-role \
+		topolvm-csi-resizer-role \
+		topolvm-csi-snapshotter-role \
+		topolvm-system:controller \
+		topolvm-system:node
+	$(KUBECTL) -n kube-system delete role \
+		topolvm-cert-manager-cainjector:leaderelection \
+		topolvm-cert-manager:leaderelection
+	$(KUBECTL) -n kube-system delete rolebinding \
+		topolvm-cert-manager-cainjector:leaderelection \
+		topolvm-cert-manager:leaderelection
+	$(KUBECTL) delete mutatingwebhookconfiguration \
+		topolvm-cert-manager-webhook \
+		topolvm-hook
+	$(KUBECTL) delete validatingwebhookconfiguration topolvm-cert-manager-webhook
+	$(KUBECTL) delete priorityclass topolvm
+
+	# Installing the new TopoLVM
+	$(KUBECTL) -n kube-system delete pod -l component=kube-scheduler,tier=control-plane
+	$(KUBECTL) create namespace topolvm-system2
+	$(KUBECTL) label namespace topolvm-system2 topolvm.io/webhook=ignore
+	$(HELM) install --namespace=topolvm-system2 topolvm ../charts/topolvm/ -f values.yaml
+	$(KUBECTL) wait --for=condition=available --timeout=120s -n topolvm-system2 deployments/topolvm-controller
+	$(KUBECTL) wait --for=condition=ready --timeout=120s -n topolvm-system2 -l="app.kubernetes.io/component=controller,app.kubernetes.io/name=topolvm" pod
+	$(KUBECTL) wait --for=condition=ready --timeout=120s -n topolvm-system2 certificate/topolvm-mutatingwebhook
+	cp ../deploy/scheduler-config/scheduler-config.yaml $(TMPDIR)/scheduler/scheduler-config.yaml
+	docker exec -it $$(docker ps -f name=topolvm-example-control-plane --format="{{.ID}}") cp /mnt/host/scheduler/kube-scheduler.yaml /etc/kubernetes/manifests/kube-scheduler.yaml
+
+	# Migration resources
+	$(KUBECTL) -n topolvm-system2 patch ds topolvm-node -p '{"spec": {"template": {"spec": {"nodeSelector": {"unschedule": "true"}}}}}'
+	go run cmd/migration-example/main.go
+	$(KUBECTL) delete -f ../config/crd/bases/topolvm.cybozu.com_logicalvolumes.yaml
+	$(KUBECTL) delete ns topolvm-system
+	$(KUBECTL) -n topolvm-system2 patch ds topolvm-node --type json -p='[{"op": "remove", "path": "/spec/template/spec/nodeSelector/unschedule"}]'
+	$(KUBECTL) wait --for=condition=ready --timeout=120s -n topolvm-system2 -l="app.kubernetes.io/component=node,app.kubernetes.io/name=topolvm" pod
+	sleep 30
```

Comment:
You are using `kubectl wait` command before and after this line.
Is this 30 sec sleep still required?

Reply:
Thank you. I fixed to using `kubectl wait`.